src/main/webapp/index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>YouTube Lounge</title>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">
+    <link rel="stylesheet" href="index_style.css"></link>
+    <script src="index_script.js"></script>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/js/bootstrap.min.js"></script>
+  </head>
+  <body >
+      
+      <!-- content that resizes when side bar opens -->
+      <div id = "main">
+           <div class= "header" >
+                <img
+                 src="https://firebasestorage.googleapis.com/v0/b/codesign-prod.google.com.a.appspot.com/o/user_assets%2FlyxSOzzXdt1gu3M1bgLd%2FfGwBG6ucZZcvCT6IQ54Y%2Foriginal.svg?alt=media&token=2e67a119-d7c8-4cbb-91de-04f256f74609"
+                class = "YTimage" />
+                <span class = "title">Lounge</span>
```

Comment:
Can we change the title to "YouTube Lounge" instead?

Reply:
I used the youtube logo, its the image right above it

Reply:
or do you mean change the class name?

Reply:
My apologies, I misread the file and assumed you were creating a title element which in that case I wanted it to read as "YouTube Lounge" instead, but you already have that. Sorry!